src/epd5in65f/mod.rs

```diff
@@ -229,11 +233,8 @@ where
         self.interface.cmd_with_data(spi, command, data)
     }
 
-    fn wait_busy_high(&mut self) {
-        self.interface.wait_until_idle(true);
-    }
-    fn wait_busy_low(&mut self) {
-        self.interface.wait_until_idle(false);
+    fn wait_busy_low(&mut self, delay: &mut DELAY) {
+        self.interface.wait_until_idle(delay, false);
```

Comment:
```suggestion
        self.interface.wait_until_idle(delay, !IS_BUSY_LOW);
```
Alternatively we could remove the IS_BUSY_LOW constant here and just use true and false for `ẁait_until_idle` and `wait_busy_low` to be a bit more consistent.

Reply:
Since its not that important, I'm just gonna merge this now.